arrow-libs/optics/arrow-optics/src/commonMain/kotlin/arrow/optics/dsl/at.kt

```diff
@@ -10,38 +10,28 @@ import arrow.optics.typeclasses.At
  * DSL to compose [At] with a [Lens] for a structure [S] to focus in on [A] at given index [I].
  *
  * @receiver [Lens] with a focus in [S]
- * @param AT [At] instance to provide a [Lens] to zoom into [S] at [I]
+ * @param at [At] instance to provide a [Lens] to zoom into [S] at [I]
  * @param i index [I] to zoom into [S] and find focus [A]
  * @return [Lens] with a focus in [A] at given index [I].
  */
-public fun <T, S, I, A> Lens<T, S>.at(AT: At<S, I, A>, i: I): Lens<T, A> = this.compose(AT.at(i))
-
-/**
- * DSL to compose [At] with a [Prism] for a structure [S] to focus in on [A] at given index [I].
- *
- * @receiver [Prism] with a focus in [S]
- * @param AT [At] instance to provide a [Lens] to zoom into [S] at [I]
- * @param i index [I] to zoom into [S] and find focus [A]
- * @return [Optional] with a focus in [A] at given index [I].
- */
-public fun <T, S, I, A> Prism<T, S>.at(AT: At<S, I, A>, i: I): Optional<T, A> = this.compose(AT.at(i))
+public fun <T, S, I, A> Lens<T, S>.at(at: At<S, I, A>, i: I): Lens<T, A> = this.compose(at.at(i))
```

Comment:
A `Lens` will be an `Optional`, right? So can we omit this syntax like we do for `nullable`?

Reply:
Not really, because the result for `at.at(i)` is a `Lens`. So if we remove this overload we would be losing a bit of information.